src/main/scala/Numbers.sc

```diff
@@ -0,0 +1,25 @@
+
+l_scale("1234567891111")
+s_scale("1234567891111")
+
+
+def s_scale(n: String): String = n match {
+  case "" => ""
+  case n if n.length < 4 => n.toString
+  case n if n.length < 7 => n.substring(0,n.length-3)+ " thousand and " + s_scale(n.substring(n.length-3,n.length))
+  case n if n.length < 10 => n.substring(0,n.length-6)+ " million, " + s_scale(n.substring(n.length-6,n.length))
+  case n if n.length < 13 => n.substring(0,n.length-9)+ " billion, " + s_scale(n.substring(n.length-9,n.length))
+  case n if n.length < 16 => n.substring(0,n.length-12)+ " trillion, " + s_scale(n.substring(n.length-12,n.length))
+
+  case n => ""
+}
+def l_scale(n: String): String = n match {
```

Comment:
this method is the same as "s_scale" but you're just inserting the different word, you could rework it so that you only have one method and based on which scale you want it would insert a particular word, or something